source/gui/settingsDialogs.py

```diff
@@ -129,80 +133,82 @@ class GeneralSettingsDialog(SettingsDialog):
 	)
 
 	def makeSettings(self, settingsSizer):
-		languageSizer=wx.BoxSizer(wx.HORIZONTAL)
+		settingsSizerHelper = guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
+		self.languageNames = languageHandler.getAvailableLanguages()
+		languageChoices = [x[1] for x in self.languageNames]
 		# Translators: The label for a setting in general settings to select NVDA's interface language (once selected, NVDA must be restarted; the option user default means the user's Windows language will be used).
-		languageLabel=wx.StaticText(self,-1,label=_("&Language (requires restart to fully take effect):"))
-		languageSizer.Add(languageLabel)
-		languageListID=wx.NewId()
-		self.languageNames=languageHandler.getAvailableLanguages()
+		languageLabelText = _("&Language (requires restart to fully take effect):")
 		# Translators: The list of languages for NVDA.
-		self.languageList=wx.Choice(self,languageListID,name=_("Language"),choices=[x[1] for x in self.languageNames])
+		languageCtrlName = _("Language")
+		self.languageList=settingsSizerHelper.addLabeledControl(languageLabelText, wx.Choice, name=languageCtrlName, choices=languageChoices)

```

Comment:
the name=languageControlName for wx.Choice is not needed (assuming it is never shown visually).